examples/CRISP/enclave.config.yaml

```diff
@@ -0,0 +1,31 @@
+chains:
+  - name: "hardhat"
+    rpc_url: "ws://localhost:8545"
+    contracts:
+      enclave: "0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0"
+      ciphernode_registry: "0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9"
+      filter_registry: "0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9"
```

Comment:
Wrong contract addresses, the default ones everywhere else are
```yml
    contracts:
      enclave: "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512"
      ciphernode_registry: "0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0"
      filter_registry: "0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9"
```

Reply:
Hey I am not going to fix this  instead I think it is better we do that in a different PR